db.js

```diff
@@ -0,0 +1,33 @@
+const db_File_Name = "tasksFile.JSON"
+fs = require('fs')
+
+function ReadData(){
+    if (fs.existsSync(db_File_Name))
+    {
+        return fs.readFileSync(db_File_Name)
+    }
+    else
+    {
+        initial_data = "{}"
+        fs.writeFileSync(db_File_Name, initial_data)
+        return initial_data
+    }
+}
+
+function WriteData(data){
+    var succeeded_written_to_file = true
+
+    fs.writeFile(db_File_Name, data, function(err){
```

Comment:
is it sync or async? if it async you will always return true

THIS IS IMPORTANT TO UNDERSTAND - IF YOU DON'T CALL ME